### ISSUE TITLE
Améliorer l’esthétique de la page d’accueil du site archeo-lex.fr

### ISSUE THREAD

**Issue by Seb35** [Legilibre/Archeo-Lex#27]
La page d’accueil du site https://archeo-lex.fr est très rudimentaire. Il faudrait la rendre un peu plus attractive d’une part et la mettre aux couleurs de Légilibre d’autre part (c’est-à-dire reprendre la trame de http://legilibre.fr et http://anomalies.legilibre.fr).

**Comment by fgallaire** [comment]
@Seb35 je ne pense pas qu'il faille maintenir la page (c'est une réflexion globale et tu fais ce que tu veux bien sûr).

*Archéo Lex* est un logiciel libre, la page GitHub + les informations Wiki suffisent, comme pour *legi.py*.

La page *anomalies* est un service produit par *legi.py*. La page équivalente serait donc un lien vers des dépôts Git mis à jour produits par *Archéo Lex*.

**Comment by Seb35** [comment]
Effectivement, et ça peut permettre de retirer un clic, donc améliorer l’accès [en perdant moins de visiteurs].

Après, techniquement, la page d’accueil actuelle est une simple page HTML (d’ailleurs je viens d’ajouter le lien vers Légilibre au passage) mais la liste des dépôts est créée sur mon fork de GitList – disponible sur https://github.com/Seb35/gitlist/tree/archeolex (je le transfère sur Légilbre ?).

Pour modifier cette page, c’est le fichier [twig/index.twig](https://github.com/Seb35/gitlist/blob/archeolex/themes/archeolex/twig/index.twig) (bien prendre la branche 'archeolex' du dépôt). Si cette page « d’accueil » est redesignée, il faudrait probablement revoir les autres pages pour uniformiser aux couleurs de Légilibre, donc modifier le style global du fork GitList.

**Comment by fgallaire** [comment]
> (je le transfère sur Légilbre ?)

Oui il me semble que ce serait bien que tout ce qui permet de faire fonctionner les sites/applications/services soit centralisé ici.

**Comment by yohanboniface** [comment]
Retour à chaud, j'ai pas vu de lien vers https://archeo-lex.fr/ sur la home du projet Github, et donc je suis passé à côté de https://archeo-lex.fr/codes qui est pourtant une ressource intéressante. Donc je suggère d'ajouter https://archeo-lex.fr/ en lien du projet (ce que je ne peux pas faire en PR). :)

Au passage, je vois pas le code des transports sur https://archeo-lex.fr/codes, une raison particulière? :)

**Comment by Seb35** [comment]
Bien vu pour le site de présentation, je l’ai ajouté dans l’en-tête et dans le README.

Pour le code des transport, c’est juste que je ne les ai pas tous calculés, il est en train de tourner et devrait être prêt dans quelques heures.

**Comment by Seb35** [comment]
@yohanboniface Ajouté le code des transports !

**Comment by Seb35** [comment]
Je viens d’améliorer la page d’accueil. En relisant les discussions ci-dessus, je pense qu’il est quand même préférable d’avoir une page d’accueil un peu explicative, on trouve ça souvent aujourd’hui sur les sites : une présentation avant de rentrer dans le dur.

C’est ce que j’ai essayé de faire, notamment avec la 2e partie [Fonctionnement en image](https://archeo-lex.fr/#fonctionnement) en guise d’introduction rapide à l’interface. La 3e partie [Présentation](https://archeo-lex.fr/#presentation) est inspirée de l’ancienne page d’accueil, mais mise à jour.

J’ai transféré à Légilibre le dépôt [Archeo-Lex-web](https://github.com/Legilibre/Archeo-Lex-web) -- il faudrait adapter le README d’ailleurs.

Bien sûr, si quelqu’un a des meilleurs goûts que moi en matière de couleurs et des idées en matière d’ergonomie et de présentation, je suis preneur.
